Corale.Colore/Core/Device.cs

```diff
@@ -40,7 +40,7 @@ public abstract class Device : IDevice
         /// <summary>
         /// Clears the current effect on the device.
         /// </summary>
-        public void Clear()
+        public virtual void Clear()
         {
             SetAll(Color.Black);
```

Comment:
The `SetAll` method of clearing is the only one that is device-agnostic (every device has a "different" implementation of the `None` effect while every device will have to implement a `SetAll` method).

Short of marking it abstract which would force every device to explicitly make an implementation which might not be necessary, `SetAll` is the only workable fallback for clearing.